lib/datasource/remote/api/http_repo/seeds_chain_names.dart

```diff
@@ -0,0 +1,26 @@
+const String _chainTelos = 'telos';
```

Comment:
Lower case "telos" makes sense but we have a conflict with legacy code in `seeds.tokensmaster.cpp` and the existing `tmastr.seeds` table entries which use "Telos".

Suggested fix would be to patch a conditional expression here which converts "Telos" to "telos". https://github.com/JoinSEEDS/seeds_light_wallet/blob/ac72252fd2ee92a5517336d7a8faaaa36a88d8f1/lib/datasource/remote/model/token_model.dart#L93

Reply:
Is this an actual bug or ... ?

Reply:
We need to choose whether "telos" or "Telos" is the string we use in TokenModel.chainName.
1. If "Telos" (capitalized), change https://github.com/JoinSEEDS/seeds_light_wallet/blob/0e0cdfd1597de0e616c03d875255317c05377ec3/lib/datasource/remote/api/http_repo/seeds_chain_names.dart#L1
2. If "telos" (lowercase), make the patch so that legacy metadata containing "Telos" is normalized.

So yes, it's a bug that will appear as soon as we start using this PR code.